ui/modal_templates.go

```diff
@@ -59,14 +65,32 @@ type modalLoad struct {
 }
 
 func (win *Window) LoadModalTemplates() *ModalTemplate {
+	editorBoarderColor := color.NRGBA{41, 112, 255, 255}
+	walletName := win.theme.Editor(new(widget.Editor), "")
+	walletName.LineColor = editorBoarderColor
+	walletName.TitleLabelColor = editorBoarderColor
```

Comment:
Why is this done repeatedly? Isn't the inactive border colour the default editor colour until it is active or there's an error?